src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -20,8 +26,9 @@ public static partial class FrameworkInfo
         /// CLR 4 => .NET 4.0, 4.5.x, 4.6.x, 4.7.x
         /// </remarks>
         /// <param name="clrVersion">The CLR version: 1, 2 or 4</param>
-        /// <returns>The framework installation or null if none is found.</returns>
-        public static FrameworkInstallation? GetLatest(int clrVersion)
+        /// <param name="registryLock">Blocks this function to access the System's registry.</param>
+        /// <returns>The framework installation or null if none is found, used for fine-tuning the Latest info.</returns>
+        public static FrameworkInstallation? GetLatest(int clrVersion, bool registryLock = false)
```

Comment:
Again, can we just have the function return `null` in case registry is inaccessible? Why do we need a shared property and a method parameter for that?